### PR TITLE
Apple Watch Series 4 Complications

### PR DESCRIPTION
**Pull request by apabari**
I've made some modifications to allow usage of the complication on the new Series 4 watch:
--> .graphicCircular complication (full ring, glucose in middle, trend below)
--> .graphicCorner complication (outer text: glucose and trend, inner text: time)

I know this isn't fully aligned to the suggestions in #803 -- but please bear with me, as I'm new to Swift.  For anyone with a Series 4 watch, this is just a short-term 'enhancement' to allow use of the Loop complication on one of the new watch faces.

### CODE REVIEW

**Comment by elnjensen** [comment]
@apabari   Thanks for working on this!  It's something I've wanted to have, but haven't had time to work on.  One thing I see is that your pull request includes a number of files that aren't actually part of the code changes you're working on (a number of Info.plist files, Cartfile, Cartfile.resolved).  Can you remove those from your PR?  You might need to start with a clean copy of the dev branch (and possibly a new pull request).  I'm not sure if there's an easy way to drop those files if they aren't in separate commits from the ones that actually have the new Watch changes, though.   But it's definitely useful to see these complications as a short-term enhancement for Series 4 watch users.

File: WatchApp Extension/Info.plist [hunk 1]
```diff
@@ -26,13 +26,17 @@
 	<string>$(PRODUCT_MODULE_NAME).ComplicationController</string>
 	<key>CLKComplicationSupportedFamilies</key>
 	<array>
-		<string>CLKComplicationFamilyModularSmall</string>
 		<string>CLKComplicationFamilyCircularSmall</string>
 		<string>CLKComplicationFamilyExtraLarge</string>
-		<string>CLKComplicationFamilyUtilitarianSmallFlat</string>
-		<string>CLKComplicationFamilyUtilitarianLarge</string>
+		<string>CLKComplicationFamilyGraphicBezel</string>
```
Comment (elnjensen) [inline WatchApp Extension/Info.plist]: Looks like maybe you've also enabled the Graphic Rectangular and Graphic Bezel complications in settings?  Probably should leave those unchecked until there's code to support them.

Comment (apabari) [reply WatchApp Extension/Info.plist]: Thanks!  Good point.  I was working on complications for the Graphic Rectangular and Graphic Bezel, but haven't completed them yet; I should have unchecked them.

**Comment by elnjensen** [comment]
I played around with coloring a little bit.  It looks like only the inner text provider supports color for the [stacked text Graphic Corner complication](https://developer.apple.com/documentation/clockkit/clkcomplicationtemplategraphiccornerstacktext).  I got it to work by assigning the `tintColor` to the `textProvider` *before* assigning that `textProvider` instance to the complication.  This also works for changing the ring color based on time for the Graphic Circular.

**Comment by elnjensen** [comment]
@apabari   Anything I can do to help move this along?  This is great work and I don't want to get in your way, but I think this PR is unlikely to move forward until it's slimmed down to only the files that deal with the complications.

**Comment by apabari** [comment]
@elnjensen, Sorry for the delayed response.  I haven't had any time to look at this again, but I hope to do so in the next week or so.  If you have some time to correct the issues with the extra files and move this forward, I would greatly appreciate it... and I'll owe you one!

**Comment by ps2** [comment]
s4 watch complications were merged in in #848

**Pull request closed without merging**